Shell/Builtin.cpp

```diff
@@ -256,6 +256,15 @@ int Shell::builtin_dirs(int argc, const char** argv)
     return 0;
 }
 
+int Shell::builtin_exec(int argc, const char** argv)
+{
+    Vector<const char*> argv_vector;
+    argv_vector.append(argv + 1, argc - 1);
+    argv_vector.append(nullptr);
+
+    execute_process(std::move(argv_vector));
```

Comment:
just `move(argv_vector)`.

Reply:
Oh I didn't know about that, thanks. Is it worth maybe mentioning this in the CodingStyle document?

Reply:
I mean...`std::anything` is rather rare here, but mentioning it in the coding style could be useful, I suppose. 🤷